docs/topics/yaml-rule-structure-syntax.adoc

```diff
@@ -0,0 +1,520 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
+
+:_content-type: REFERENCE
+[id="yaml-rule-structure-syntax_{context}"]
+= YAML rule structure and syntax
+
+{ProductShortName} rules are written in YAML. Each rule consists of metadata, conditions and actions. It instructs the analyzer to take specified actions when given conditions match.
+
```

Comment:
"it" doesn't really fit here -- doesn't really replace "each rule." Suggestion: "...and actions, which instruct..."